crmsh/preflight_check/task.py

```diff
@@ -25,13 +25,17 @@ class Task(object):
     Task is a base class
     Use for record the information of each test case
     """
+    REBOOT_WARNING = """!!! WARNING WARNING WARNING !!!
+THIS CASE MAY LEAD TO NODE BE FENCED.
+TYPE Yes TO CONTINUE, OTHER INPUTS WILL CANCEL THIS CASE: """
```

Comment:
Just to maintain the simliar habit to most of other CLI tools. How about:
s%CASE: %CASE [Yes/No] (No): %